korman/exporter/physics.py

```diff
@@ -212,11 +212,11 @@ def _iter_object_tree(bo, stop_at_subworld):
                 bo_xformed = bo.plasma_object.has_transform_animation
 
                 # MOUL: only objects that have animation data are kPhysAnim
-                if ver != pvMoul or bo_xformed:
+                if mod.enabled and (ver != pvMoul or bo_xformed):
                     _set_phys_prop(plSimulationInterface.kPhysAnim, simIface, physical)
                 # PotS: objects inheriting parent animation only are not pinned
                 # MOUL: animated objects in subworlds are not pinned
-                if bo_xformed and (ver != pvMoul or subworld is None):
+                if (bo_xformed and (ver != pvMoul or subworld is None)) or ((ver != pvMoul) and subworld is not None and (not bo_xformed)):
```

Comment:
I suspect this is what's fixing subworlds, but the `kPinned` property only has an effect in PotS. The conditional is really hard to follow, so I'm not really able to glance in and see what's happening.